src/spyglass/position/v1/position_dlc_pose_estimation.py

```diff
@@ -406,6 +406,16 @@ def fetch_dataframe(self, *attrs, **kwargs):
             axis=1,
         )
 
+    def fetch_video_name(self):
```

Comment:
This name doesn't totally align with what's going - it fetches the path instead of the name. I think the calling func then applies `stem` to extract name from path. I think you could either change this to `fetch_video_path` or add `stem` to the func body

Reply:
changed the name. thanks